Sources/Logging/Logging.swift

```diff
@@ -142,31 +176,30 @@ extension Logging.Level: Comparable {
 extension Logging.Metadata.Value: Equatable {
     public static func == (lhs: Logging.MetadataValue, rhs: Logging.MetadataValue) -> Bool {
         switch (lhs, rhs) {
-        case (.string(let lhs), .string(let rhs)):
```

Comment:
why the `let` moving? I think they're better where they actually bind stuff rather than the outside